src/components/FinalFormComponents/Checkbox.js

```diff
@@ -0,0 +1,27 @@
+import { useField } from 'react-final-form'
+import { Checkbox as CoreCheckbox } from '@dhis2/ui-core'
+import React from 'react'
+import propTypes from 'prop-types'
+
+export const Checkbox = ({ label, name, checkedInitially: initialValue }) => {
+    const { input } = useField(name, {
```

Comment:
I missed this in the previous PRs, but I was wondering why you use `useField` here. I remember that in the readme for react-final-form-hooks it was mentioned that there are certain downsides to using the hooks (see top of readme https://github.com/final-form/react-final-form-hooks). I wonder if that still goes for the hooks here? In general maybe it's nice for us to standardise on defaulting to a certain approach? To prevent divergence between our implementations?

Reply:
It's actually explained here:
https://github.com/final-form/react-final-form-hooks#whats-the-difference-between-react-final-form-hooks-and-the-hooks-introduced-in-react-final-form-v5

After reading this, I think we want to standardize using react-final-form. We'll have all the benefits when using that lib's hooks as well, so we don't need to worry about it.

Personally I prefer the hook approach over the react component approach, it's more readable to me (and it separates setting up a field and rendering a field)

Reply:
> I think we want to standardize using react-final-form.

Yeah I think that that'd be good 👍